### DISCUSSION TITLE
站点推荐：linosx.org

### DISCUSSION THREAD

**Discussion by yangfl** [Gamuxorg/bbs#217] [建议和闲聊]
https://linosx.org/ 老毛子的游戏站，全是linux游戏，够整一阵了

**Comment by shouhuanxiaoji** [comment]
感谢推荐。
目前遇到两个问题，一是打包的人太少，更新慢，二是正在写新版网站，缺前端。
另外催一下，我在salsa.debian.org推送了一个1.6的scrcpy，麻烦老哥看看

**Comment by genufish** [comment]
https://rutracker.org/forum/viewforum.php?f=1992

我也丢一个

**Comment by SoulMelody** [comment]
https://gogfan.ru/linux/

**Comment by Leilei332** [comment]
> https://linosx.org/ 老毛子的游戏站，全是linux游戏，够整一阵了

怎么肆零肆了
